.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on: pull_request
```

Comment:
What about changing this to:

```
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

Reply:
@marmelo

Reply:
@vpereira01 @pedrorijo91 what do you think? This workflow seems ready to merge with or without this change.

Reply:
Just updated this.